http-server-netty/src/main/java/io/micronaut/http/server/netty/jackson/JsonViewCodecResolver.java

```diff
@@ -24,6 +25,7 @@
  * @author graemerocher
  * @since 1.1
  */
+@Internal
 public interface JsonViewCodecResolver {
```

Comment:
This can just be made package private. The method is already package private so it wouldn't be feasible to use the interface in any meaningful way

Reply:
which method is package-private?

Reply:
Nevermind - confused it for a class. Still though I think we can make the class package private (not saying we shouldn't add @Internal)

Reply:
it is an interface it is public

Reply:
in that case, I will also make the only implementation package-private. I assume `@Internal` is not required then either?